source/libnormaliz/HilbertSeries.h

```diff
@@ -323,30 +313,29 @@ class IntegrationData{
     const pair<HilbertSeries, mpz_class>& getWeightedEhrhartSeries() const;
     bool isPolynomialHomogeneous() const;
     mpz_class getNumeratorCommonDenom() const;
-    
+
     const vector<mpz_class>& getNum_ZZ() const;
     // returns the denominator, repr. as a map of the exponents of (1-t^i)^e
     const map<long, denom_t>& getDenom() const;
 
     const vector<mpz_class>& getCyclotomicNum_ZZ() const;
     // returns the denominator, repr. as a map of the exponents of the cyclotomic polynomials
     const map<long, denom_t>& getCyclotomicDenom() const;
-    
+
     void computeWeightedEhrhartQuasiPolynomial() const;
     bool isWeightedEhrhartQuasiPolynomialComputed() const;
-    const vector< vector<mpz_class> >& getWeightedEhrhartQuasiPolynomial() const;
+    const vector<vector<mpz_class> >& getWeightedEhrhartQuasiPolynomial() const;
```

Comment:
@w-bruns Note that we could add the option `Standard: Cpp11` to get this instead:
```suggestion
    const vector<vector<mpz_class>>& getWeightedEhrhartQuasiPolynomial() const;
```
Let me know what you prefer.

Reply:
I would not mind >>. However, do we run into problems with older compilers? We could leave > > for a while.

Reply:
Sure, let's leave it for now; I added the option but commented it out